<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbstractDialogWindow</class>
 <widget class="QDialog" name="AbstractDialogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>266</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #16485c;
color: #f0f0f0f0;

QPushButton{
	border: 2px solid black;
	border-radius: 15px;
};

QToolButton{
	border: 2px solid black;
	border-radius: 15px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>15</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
